docs/cli/cmdline/index.md

```diff
@@ -29,8 +29,6 @@ All sub-commands support the `--help` flag as well for specific details.
 
 <!-- - `--disable-check-git-untracked`      Disable git check for untracked files. -->
 <!-- - `--disable-check-git-uncommitted`    Disable git check for uncommitted files. -->
-<!-- - `--disable-checkpoint`               Disable checkpoint checks for updates. -->
```

Comment:
They should stay documented..

Reply:
these lines are commented out in this file, they dont show in the website nor in the rendered markdown. I don't know why they are commented.

The flags are still documented here: 
- https://terramate.io/docs/cli/cmdline/version
- https://github.com/terramate-io/terramate/blob/main/docs/cli/cmdline/version.md